dashboard/app/models/user.rb

```diff
@@ -332,7 +332,8 @@ def district_teachers(cohort = nil)
   validates :name, length: {within: 1..70}, allow_blank: true
   validates :name, no_utf8mb4: true
 
-  validates :age, presence: true, on: :create # only do this on create to avoid problems with existing users
+  is_google = proc {|user| user.provider == 'google_oauth2'}
+  validates :age, presence: true, on: :create, unless: is_google # only do this on create to avoid problems with existing users
```

Comment:
nit: This line is over 100 characters. And the comment should probably be updated.